README.md

```diff
@@ -66,14 +66,15 @@ applicationUser: # optional - populate if creating test data for users other tha
   tenantId: SPECFLOW_POWERAPPS_TENANTID optional # mandatory
   clientId: SPECFLOW_POWERAPPS_CLIENTID # mandatory
   clientSecret: SPECFLOW_POWERAPPS_CLIENTSECRET # mandatory
-users: # mandatory
+users: # One user is mandatory, additional users are optional
```

Comment:
The comment is about the config file schema so best to keep this as-is because the `users` array is mandatory.

```suggestion
users: # mandatory
```